app/src/main/java/com/fstyle/androidtrainning/screen/main/MainActivity.java

```diff
@@ -8,12 +8,17 @@
 import android.view.Menu;
 import android.view.MenuItem;
 import com.fstyle.androidtrainning.R;
+import com.fstyle.androidtrainning.application.MainApplication;
+import com.fstyle.androidtrainning.data.remote.service.config.LastFmApi;
+import com.fstyle.androidtrainning.model.Track;
 import com.fstyle.androidtrainning.screen.BaseActivity;
 import com.roughike.bottombar.BottomBar;
 import com.roughike.bottombar.OnTabSelectListener;
+import java.util.List;
```

Comment:
ERROR: Unused import: java.util.List. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports